_config.yml

```diff
@@ -26,7 +26,13 @@ keep_files: [".git", ".svn", "wiki-images", "images", "downloads"]
 lsi:  false # don't produce an index for related posts
 safe: true  # disables plugins
 
-markdown: kramdown
+markdown: redcarpet
+
+redcarpet:
+    extensions:
+        - prettify
+        - fenced_code_blocks
```

Comment:
If that default behaviour changes, specifying it here makes it clear that we explicitly want it. I'll add a comment to explain this.